src/render/painter.js

```diff
@@ -753,7 +754,8 @@ class Painter {
         const terrain = this.terrain && !this.terrain.renderingToTexture; // Enables elevation sampling in vertex shader.
         const rtt = this.terrain && this.terrain.renderingToTexture;
         const fog = this.style && this.style.fog;
-        const fogOpacity = fog && fog.getFogPitchFactor(this.transform.pitch);
+        const fogAlpha = (fog && fog.properties && fog.properties.get('color').a) || 1.0;
```

Comment:
Above you have `fog && fog.properties && fog.properties.get(...)` but here only `fog && fog.properties.get(...)`. Can this be made consistent/minimal?